thijsvanloef/palworld-server-docker
# Change Docusaurus to the more easily findable through search engines

## Context 
Underperforming findability via search engines, as a result of wierd trailing slash handling from github pages.


## Choices

Missing "how to update" guide
Group Cronjobs to a folder
Change Github Pages for CloudFlare pages.

## Test instructions

1. Go to https://palworld-server-docker.loef.dev/ and check that it is hosted on Cloudflare

## Checklist before requesting a review
